src/components/templates/TemplateInventory.vue

```diff
@@ -91,6 +91,7 @@
                 <AtomTextarea
                     :value="props.row.Comments"
                     class="comment-width"
+                    :maxlength="1000"
```

Comment:
Technically, you can also use camelCase when passing props to a child component (except in [DOM templates](https://vuejs.org/guide/essentials/component-basics.html#dom-template-parsing-caveats)). However, the convention is using kebab-case in all cases to align with HTML attributes:
https://vuejs.org/guide/components/props.html#prop-passing-details